impeller/entity/shaders/tiled_texture_fill.frag

```diff
@@ -12,11 +12,11 @@ uniform f16sampler2D texture_sampler;
 uniform FragInfo {
   float16_t x_tile_mode;
   float16_t y_tile_mode;
```

Comment:
We don't used the tiled texture shader on vulkan, but we still need to move the alpha so that this can share the same vertex shader as texture.frag.